assets/src/scss/style.scss

```diff
@@ -28,7 +28,6 @@ Text Domain: planet4-master-theme
 @import "base/shame";
 
 // Components
-@import "components/blockquote";
```

Comment:
Since the Quote is a core WordPress block, I thought it made more sense to move this to the `core-overrides` folder